axelrod/match.py

```diff
@@ -159,6 +159,12 @@ def normalised_cooperation(self):
         """Returns the count of cooperations by each player per turn"""
         return iu.compute_normalised_cooperation(self.result)
 
+    def state_distribution(self):
+        return iu.compute_state_distribution(self.result)
```

Comment:
Could we have a brief docstring for these two methods (similar to the ones in the interactions_utils).